### PR TITLE
Update CDL crop group names

### PR DESCRIPTION
**Pull request by caseycesari**
## Overview

Updated per client document.

Regenerate derivative files.

Connects to #218 

### Demo

image

## Testing Instructions

- Verify that the values in the legend match those specified in https://docs.google.com/spreadsheets/d/1uA4vXIvQvS3HFqXLamVYPhhPVp4P-xyPvzbG8QUBe3Q/edit?ts=591e024b#gid=0.

### CODE REVIEW

File: src/icp/js/src/core/cropTypes.json [hunk 1]
```diff
@@ -3,52 +3,62 @@
...
     "11":"Grassland/Pasture",
     "12":"Buckwheat",
-    "13":"Root Vegs",
-    "14":"Solanums",
-    "15":"Other Crops",
-    "16":"Vegs & Fruits",
+    "13":"Root Vegetables",
+    "14":"Solanum Crops",
+    "15":"Crops (other)",
+    "16":"Miscellaneous Vegetables and Fruits",
```
Comment (arottersman) [inline src/icp/js/src/core/cropTypes.json]: @ajrobbins This guy is too long now. Can we compromise on `Misc. Vegetables and Fruits`
screen shot 2017-07-12 at 3 10 10 pm

File: src/icp/js/src/core/cropTypes.json [hunk 2]
```diff
@@ -3,52 +3,62 @@
...
     "48":"Strawberries",
-    "49":"Cucurbits",
-    "5":"Flowers",
+    "49":"Cucurbits (other)",
+    "5":"Flower Crops",
     "50":"Pumpkins",
     "51":"Blueberries",
     "52":"Raspberry",
+    "53":"Almond with cover crop",
+    "54":"Apple with cover crop",
```
Comment (arottersman) [inline src/icp/js/src/core/cropTypes.json]: Unfortunately this process involves manually removing all these cover crops from this json

File: src/icp/js/src/core/cropTypes.json [hunk 3]
```diff
@@ -3,52 +3,62 @@
...
+    "55":"Blueberry with cover crop",
+    "56":"Cherry with cover crop",
+    "57":"Raspberry with cover crop",
+    "58":"Watermelon with cover crop",
+    "59":"Pumpkin with cover crop",
     "6":"Tobacco",
+    "60":"Wildflower",
+    "61":"Woody",
+    "62":"Mix",
```
Comment (arottersman) [inline src/icp/js/src/core/cropTypes.json]: And also these three (60-62) which are pollinator plantings (enhancements)

File: deployment/ansible/group_vars/all [hunk 4]
```diff
@@ -30,7 +30,7 @@ nodejs_npm_version: 2.1.17
 
 apache_version: "2.4.7-*"
 
-java_version: "7u121-*"
+java_version: "7u131-*"
```
Comment (arottersman) [inline deployment/ansible/group_vars/all]: What does this project use java for? Can we maybe remove this and the associated `azavea.java` role? @mmcfarland

File: src/icp/js/src/core/cropTypes.json (hunk 1 above)
Comment (ajrobbins) [inline src/icp/js/src/core/cropTypes.json]: Yes, that should be fine.

Comment (caseycesari) [inline src/icp/js/src/core/cropTypes.json]: Cool. I'll update it.

File: src/icp/js/src/core/cropTypes.json (hunk 2 above)
Comment (caseycesari) [inline src/icp/js/src/core/cropTypes.json]: Ah okay. I'll remove them.

File: deployment/ansible/group_vars/all (hunk 4 above)
Comment (caseycesari) [inline deployment/ansible/group_vars/all]: I wasn't able to provision the project without upgrading this. Not sure how it's utilizied.

File: src/icp/js/src/core/cropTypes.json (hunk 3 above)
Comment (caseycesari) [inline src/icp/js/src/core/cropTypes.json]: Thanks.

**Comment by caseycesari** [comment]
Pushed a fixup commit.

**Comment by caseycesari** [comment]
Thanks!

**Pull request merged**